identity/app/views/upsell/upsellContainer.scala.html

```diff
@@ -40,24 +40,14 @@
     }
 }
 
-
-@prefill = @{
-    Json.toJson(Map(
-        "csrfToken" -> play.filters.csrf.CSRF.getToken.map(_.value),
-        "accountToken" -> requestHeader.cookies.get("SC_GU_GUEST_PW_SET").map(_.value),
-        "email" -> Option(email),
-        "hasPassword" -> Option(hasPassword.toString),
-        "hasSocialLinks" -> Option(hasSocialLinks.toString)
-    ).collect{
-        case (name, Some(value)) => name -> value
-    }
-    ).toString()
-}
-
-<div class="js-identity-block-list" data-page-variant="@{pageVariant.jsName}">
-    <script type="text/json" data-for-prefill="true">
-        @Html(prefill)
-    </script>
+<div class="js-identity-block-list"
+    data-page-variant="@{pageVariant.jsName}"
```

Comment:
including this data as data attributes made their deserialisation to flow types easier (@Ap0c)